server.js

```diff
@@ -131,18 +131,59 @@ app.put('/api/v1/project/:id', (request, response) => {
   }
```

Comment:
the "if" statement on 129 should be moved above the database (on line 127)  because if there is no updated name then it should return and not query the database

Reply:
Okay both PUT for project and palettes have been fixed and tested in Postman and work as expected.